internal/metrics/metricsManager.go

```diff
@@ -132,7 +132,7 @@ func NewMetricsManager(registry prometheus.Registerer) *MetricsManager {
 			Help:      counterDescriptionOperationDurationMiliSeconds,
 			Buckets: []float64{100, 200, 300, 400, 500, 600, 700, 800, 900, 1000, 1200,
 				1400, 1600, 1800, 2000, 2400, 2800, 3200, 3600, 4000, 5000},
-		}, []string{operation}),
+		}, []string{openshiftVersionLabel, operation}),
```

Comment:
you should provide the openshift version in [line#215](https://github.com/openshift/assisted-service/pull/870/files#diff-6d51a205c8e038996615c6ed4baa7cb8822fe75ef259ad398f66be80097f8600R215)